src/helpers.js

```diff
@@ -24,22 +24,34 @@ export const downloadAddon = async (url) => {
 
   console.log('LOG: Addon temp path:', tempPath);
 
-  const addonZipPath = path.join(tempPath, `${hash}.zip`);
+  let addonFilePath = path.join(tempPath, `${hash}.zip`);
+  let addonPath = path.join(tempPath, hash);
 
-  if (!fs.existsSync(addonZipPath)) {
+  if (url.endsWith('zip')) {
+    console.log('LOG: ZIP file');
+  }
+  if (url.endsWith('crx')) {
```

Comment:
cannot we donwload chrome extension in a zip format?

Reply:
Deleted